### PR TITLE
Reissner-Nordstorm metric to symbolic module

### LINKED ISSUE #309: Add loads of predefined metrics in symbolic module
🐞 **What bugs you?**
  - Have predefined metrics in *symbolic* module

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

### PR DESCRIPTION
**Pull request by obi-wan-shinobi**
Added the Reissner-Nordstorm metric to vacuum_solutions.py and added a new test function in the test_all.py for the same.
Fixes #309 

@ritzvik 
@shreyasbapat 

Sorry about a new PR. I was trying to squash my commits in the old PR and ended up breaking everything. I had to delete my repository and refork this to my account. Starting afresh again :(

### CODE REVIEW

**Comment by ritzvik** [comment]
I’m very sorry to say that KerrNewman is defined wrongly. If you can correct it, it would be good, otherwise I’ll take care of it myself.

**Comment by obi-wan-shinobi** [comment]
> I’m very sorry to say that KerrNewman is defined wrongly. If you can correct it, it would be good, otherwise I’ll take care of it myself.

Yes. I was confused about the diagonal matrix and the first term not having an rQsq element. I'll see to it :)

**Comment by obi-wan-shinobi** [comment]
@ritzvik 
@shreyasbapat 

I've made changes and corrected the vacuum_solutions.py.
Please look into the docs error raised by @pep8speaks 
I've reformatted the code using tox. If I remove those changes to compensate for the error raised by @pep8speaks the quality check fails. If I reformat the code using tox, @pep8speaks raises error. Please assist me in removing the errors :(

**Comment by ritzvik** [comment]
@pep8speaks is not a big concern

File: src/einsteinpy/symbolic/predefined/vacuum_solutions.py [hunk 1]
```diff
@@ -55,13 +55,11 @@ def Kerr(c=constants.c, sch=symbols("r_s"), a=symbols("a")):
...
-            (r ** 2 + a ** 2 + (sch * r * (a ** 2) * (sin(theta) ** 2) / Sigma))
-            * (sin(theta) ** 2)
-        )
-        / c2,
+        (Delta * a ** 2 * sin(theta) ** 2 - (r ** 2 + a ** 2) ** 2)
+        * sin(theta) ** 2
+        / (Sigma * c2),
     ).tolist()
-    metric[0][3] = metric[3][0] = sch * r * a * (sin(theta) ** 2) / (Sigma * c)
+    metric[0][3] = metric[3][0] = 2 * sch * r * a * (sin(theta) ** 2) / (Sigma * c)
```
Comment (ritzvik) [inline src/einsteinpy/symbolic/predefined/vacuum_solutions.py]:
```suggestion
    metric[0][3] = metric[3][0] = sch * r * a * (sin(theta) ** 2) / (Sigma * c)
```

2 in the expression is coming from both `d(phi)dt` and `dtd(phi)`. So in a tensorial form, we have to redistribute the 2 again.

File: src/einsteinpy/symbolic/predefined/vacuum_solutions.py [hunk 2]
```diff
@@ -55,13 +55,11 @@ def Kerr(c=constants.c, sch=symbols("r_s"), a=symbols("a")):
...
         -Sigma / (Delta * c2),
         -Sigma / c2,
-        -(
-            (r ** 2 + a ** 2 + (sch * r * (a ** 2) * (sin(theta) ** 2) / Sigma))
-            * (sin(theta) ** 2)
-        )
-        / c2,
+        (Delta * a ** 2 * sin(theta) ** 2 - (r ** 2 + a ** 2) ** 2)
+        * sin(theta) ** 2
+        / (Sigma * c2),
```
Comment (ritzvik) [inline src/einsteinpy/symbolic/predefined/vacuum_solutions.py]: I am pretty confident about the Kerr expression. It is directly copied from wikipedia(not that it's a very reliable source, but works fine) :D 

Please revert the changes here. Or leave it if your expression reduces to the same expression( BIG IF )

File: src/einsteinpy/symbolic/predefined/vacuum_solutions.py (hunk 1 above)
Comment (ritzvik) [reply src/einsteinpy/symbolic/predefined/vacuum_solutions.py]: Also, metric tensor in classical GR is symmetric. In fact most of the tensors in physics is symmetric

**Comment by ritzvik** [comment]
@obi-wan-shinobi Thanks a lot!
I really appreciate the efforts you did in cleaning up the embarrassment that was `KerrNewman`.

I'm gonna merge :)

**Comment by ritzvik** [comment]
One thing, 
please open new branches for separate PRs, never make a PR from master.
